subset/network/Dockerfile.test_dhcplong

```diff
@@ -0,0 +1,9 @@
+FROM daq/aardvark:latest
```

Comment:
These should all be in the same docker container... They are 80% the same thing. So, one docker container, and then have one top-level test file that calls the network_tests.py helper three times.  Will dramatically cut down the amount of repetition in the files!  (e.g. "test_dhcplong_send" and "test_net_app_send" are virtually identical).

Reply:
Same docker container now being used